_includes/moto.md

```diff
@@ -0,0 +1,90 @@
+{% if include.api_resource == "creditcard" %}
+    {% assign api_resource_field_name = "payment" %}
+{% else %}
+    {% assign api_resource_field_name = "paymentorder" %}
+{% endif %}
+
+## MOTO
+
+{% include alert-agreement-required.md %}
+
+MOTO (Mail Order / Telephone Order) is a purchase where you as a merchant enter 
+the customer card details in order to make a payment. The card details are given
+over telephone or in writing, and then entered into the payment interface by the
+sales representative. Common use-cases are travel or hotel bookings, where the 
+customer calls the sales representative to make a booking. This feature is only 
+supported in the `Purchase` operation. See the abbreviated example below on how to
+implement MOTO by setting the `generatedMotoPayment` to `true`.
+ 
+
+{:.code-view-header}
+**Request**
+
+```http
+POST /psp/{{ include.api_resource }}/payments HTTP/1.1
+Host: {{ page.api_host }}
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "{{api_resource_field_name}}": {
+        "operation": "Purchase",
+        "intent": "Authorization",
+        "generateMotoPayment": true,
+        "currency": "NOK",
+        "amount": 1500,
+        "vatAmount": 0,
+        "description": "Test Moto",
+        "userAgent": "Mozilla/5.0...",
+        "language": "nb-NO",
+        "urls": {
+            "callbackUrl": "https://example.com/payment-callback"
+        },
+        "payeeInfo": {
+            "payeeId": "{{ page.merchant_id }}",
+            "payeeReference": "CD1234",
+            "payeeName": "Merchant1",
+            "productCategory": "A123",
+            "orderReference": "or-12456",
+            "subsite": "MySubsite"
+        }
+    }
+}
+```
+
+{:.code-view-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
```

Comment:
I don't quite understand what should happen after this response has been received. Where should the card details given from the payer over telephone be entered by the merchant employee? Is there a redirect URL to look for in the response that should be presented to the merchant employee or what?

Reply:
Yes! The merchant should use the redirect-operation.

Reply:
Then we need to provide that operation in the example and explain what to do with it. :)

Reply:
@sigridbra can you add en explanation in this PR ?

Reply:
This would be just as a normal purchase, the only difference being that the details are filled in by the sales representative.